eng/common/scripts/git-branch-push.ps1

```diff
@@ -60,12 +65,14 @@ if ($LASTEXITCODE -ne 0)
     exit $LASTEXITCODE
 }
 
-Write-Host "git checkout -b $PRBranchName"
-git checkout -b $PRBranchName
-if ($LASTEXITCODE -ne 0)
-{
-    Write-Error "Unable to create branch LASTEXITCODE=$($LASTEXITCODE), see command output above."
-    exit $LASTEXITCODE
+if (-not $SkipCheckout) {
```

Comment:
I'd prefer we not have the skipcheckout option but if we do need it for some reason we need to add some validation to ensure we have already checked out the specific branch otherwise some assumptions could be broken elsewhere in the script.